glyphs2fontir/src/source.rs

```diff
@@ -94,21 +100,32 @@ impl GlyphsIrSource {
     // When things like upem may have changed forget incremental and rebuild the whole thing
     fn global_rebuild_triggers(
         &self,
-        _root_dict: &HashMap<String, Plist>,
+        root_dict: &HashMap<String, Plist>,
+        raw_plist: &str,
     ) -> Result<StateSet, Error> {
-        todo!()
+        // Naive mk1: if anything other than glyphs and date changes do a global rebuild
+        // TODO experiment with actual glyphs saves to see what makes sense
+        let mut state = StateSet::new();
+        for (key, plist) in root_dict {
+            if key == "glyphs" || key == "date" {
```

Comment:
this is probably too coarse / too sensitive to changes, but I'm ok to keep like that until we have a better idea of what can change globally that would/would not affect single glyphs IR.

Reply:
SGTM